Started GET "/" for 172.18.0.1 at 2018-03-25 21:45:12 +0000
  
PG::ConnectionBad (could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
):
  
pg (0.21.0) lib/pg.rb:56:in `initialize'
pg (0.21.0) lib/pg.rb:56:in `new'
pg (0.21.0) lib/pg.rb:56:in `connect'
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:671:in `connect'
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:217:in `initialize'
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `new'
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:729:in `new_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:773:in `checkout_new_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:752:in `try_to_checkout_new_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:713:in `acquire_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:490:in `checkout'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:364:in `connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:883:in `retrieve_connection'
activerecord (5.0.6) lib/active_record/connection_handling.rb:128:in `retrieve_connection'
activerecord (5.0.6) lib/active_record/connection_handling.rb:91:in `connection'
activerecord (5.0.6) lib/active_record/migration.rb:559:in `connection'
activerecord (5.0.6) lib/active_record/migration.rb:546:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/callbacks.rb:38:in `block in call'
activesupport (5.0.6) lib/active_support/callbacks.rb:97:in `__run_callbacks__'
activesupport (5.0.6) lib/active_support/callbacks.rb:750:in `_run_call_callbacks'
activesupport (5.0.6) lib/active_support/callbacks.rb:90:in `run_callbacks'
actionpack (5.0.6) lib/action_dispatch/middleware/callbacks.rb:36:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/remote_ip.rb:79:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/debug_exceptions.rb:49:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.6) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.6) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:69:in `block in tagged'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:69:in `tagged'
railties (5.0.6) lib/rails/rack/logger.rb:24:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.4) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.6) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.4) lib/rack/sendfile.rb:111:in `call'
railties (5.0.6) lib/rails/engine.rb:522:in `call'
puma (3.11.3) lib/puma/configuration.rb:225:in `call'
puma (3.11.3) lib/puma/server.rb:624:in `handle_request'
puma (3.11.3) lib/puma/server.rb:438:in `process_client'
puma (3.11.3) lib/puma/server.rb:302:in `block in run'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `call'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `block in spawn_thread'
Started GET "/" for 172.18.0.1 at 2018-03-25 21:58:51 +0000
  
ActiveRecord::AdapterNotSpecified ('development' database is not configured. Available: ["default"]):
  
activerecord (5.0.6) lib/active_record/connection_adapters/connection_specification.rb:236:in `resolve_symbol_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/connection_specification.rb:217:in `resolve_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/connection_specification.rb:138:in `resolve'
activerecord (5.0.6) lib/active_record/connection_adapters/connection_specification.rb:168:in `spec'
activerecord (5.0.6) lib/active_record/connection_handling.rb:53:in `establish_connection'
activerecord (5.0.6) lib/active_record/railtie.rb:125:in `block (2 levels) in <class:Railtie>'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:69:in `instance_eval'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:69:in `block in execute_hook'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:60:in `with_execution_control'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:65:in `execute_hook'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:50:in `block in run_load_hooks'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:49:in `each'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:49:in `run_load_hooks'
activerecord (5.0.6) lib/active_record/base.rb:324:in `<module:ActiveRecord>'
activerecord (5.0.6) lib/active_record/base.rb:24:in `<top (required)>'
activerecord (5.0.6) lib/active_record/migration.rb:559:in `connection'
activerecord (5.0.6) lib/active_record/migration.rb:546:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/callbacks.rb:38:in `block in call'
activesupport (5.0.6) lib/active_support/callbacks.rb:97:in `__run_callbacks__'
activesupport (5.0.6) lib/active_support/callbacks.rb:750:in `_run_call_callbacks'
activesupport (5.0.6) lib/active_support/callbacks.rb:90:in `run_callbacks'
actionpack (5.0.6) lib/action_dispatch/middleware/callbacks.rb:36:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/remote_ip.rb:79:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/debug_exceptions.rb:49:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.6) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.6) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:69:in `block in tagged'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:69:in `tagged'
railties (5.0.6) lib/rails/rack/logger.rb:24:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.4) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.6) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.4) lib/rack/sendfile.rb:111:in `call'
railties (5.0.6) lib/rails/engine.rb:522:in `call'
puma (3.11.3) lib/puma/configuration.rb:225:in `call'
puma (3.11.3) lib/puma/server.rb:624:in `handle_request'
puma (3.11.3) lib/puma/server.rb:438:in `process_client'
puma (3.11.3) lib/puma/server.rb:302:in `block in run'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `call'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `block in spawn_thread'
Started GET "/" for 172.18.0.1 at 2018-03-25 21:59:49 +0000
  
URI::InvalidURIError (bad URI(is not URI?): <<default):
  
/usr/local/lib/ruby/2.2.0/uri/rfc2396_parser.rb:176:in `split'
/usr/local/lib/ruby/2.2.0/uri/rfc2396_parser.rb:211:in `parse'
activerecord (5.0.6) lib/active_record/connection_adapters/connection_specification.rb:35:in `initialize'
activerecord (5.0.6) lib/active_record/connection_adapters/connection_specification.rb:258:in `new'
activerecord (5.0.6) lib/active_record/connection_adapters/connection_specification.rb:258:in `resolve_url_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/connection_specification.rb:219:in `resolve_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/connection_specification.rb:138:in `resolve'
activerecord (5.0.6) lib/active_record/connection_adapters/connection_specification.rb:150:in `block in resolve_all'
activerecord (5.0.6) lib/active_record/connection_adapters/connection_specification.rb:149:in `each'
activerecord (5.0.6) lib/active_record/connection_adapters/connection_specification.rb:149:in `resolve_all'
activerecord (5.0.6) lib/active_record/connection_handling.rb:73:in `resolve'
activerecord (5.0.6) lib/active_record/core.rb:46:in `configurations='
activerecord (5.0.6) lib/active_record/railtie.rb:122:in `block (2 levels) in <class:Railtie>'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:69:in `instance_eval'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:69:in `block in execute_hook'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:60:in `with_execution_control'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:65:in `execute_hook'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:50:in `block in run_load_hooks'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:49:in `each'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:49:in `run_load_hooks'
activerecord (5.0.6) lib/active_record/base.rb:324:in `<module:ActiveRecord>'
activerecord (5.0.6) lib/active_record/base.rb:24:in `<top (required)>'
activerecord (5.0.6) lib/active_record/migration.rb:559:in `connection'
activerecord (5.0.6) lib/active_record/migration.rb:546:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/callbacks.rb:38:in `block in call'
activesupport (5.0.6) lib/active_support/callbacks.rb:97:in `__run_callbacks__'
activesupport (5.0.6) lib/active_support/callbacks.rb:750:in `_run_call_callbacks'
activesupport (5.0.6) lib/active_support/callbacks.rb:90:in `run_callbacks'
actionpack (5.0.6) lib/action_dispatch/middleware/callbacks.rb:36:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/remote_ip.rb:79:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/debug_exceptions.rb:49:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.6) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.6) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:69:in `block in tagged'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:69:in `tagged'
railties (5.0.6) lib/rails/rack/logger.rb:24:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.4) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.6) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.4) lib/rack/sendfile.rb:111:in `call'
railties (5.0.6) lib/rails/engine.rb:522:in `call'
puma (3.11.3) lib/puma/configuration.rb:225:in `call'
puma (3.11.3) lib/puma/server.rb:624:in `handle_request'
puma (3.11.3) lib/puma/server.rb:438:in `process_client'
puma (3.11.3) lib/puma/server.rb:302:in `block in run'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `call'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `block in spawn_thread'
Started GET "/" for 172.18.0.1 at 2018-03-25 22:00:20 +0000
  
RuntimeError (YAML syntax error occurred while parsing /app/config/database.yml. Please note that YAML must be consistently indented using spaces. Tabs are not allowed. Error: (<unknown>): did not find expected alphabetic or numeric character while scanning an anchor at line 10 column 14):
  
railties (5.0.6) lib/rails/application/configuration.rb:148:in `rescue in database_configuration'
railties (5.0.6) lib/rails/application/configuration.rb:131:in `database_configuration'
activerecord (5.0.6) lib/active_record/railtie.rb:122:in `block (2 levels) in <class:Railtie>'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:69:in `instance_eval'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:69:in `block in execute_hook'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:60:in `with_execution_control'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:65:in `execute_hook'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:50:in `block in run_load_hooks'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:49:in `each'
activesupport (5.0.6) lib/active_support/lazy_load_hooks.rb:49:in `run_load_hooks'
activerecord (5.0.6) lib/active_record/base.rb:324:in `<module:ActiveRecord>'
activerecord (5.0.6) lib/active_record/base.rb:24:in `<top (required)>'
activerecord (5.0.6) lib/active_record/migration.rb:559:in `connection'
activerecord (5.0.6) lib/active_record/migration.rb:546:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/callbacks.rb:38:in `block in call'
activesupport (5.0.6) lib/active_support/callbacks.rb:97:in `__run_callbacks__'
activesupport (5.0.6) lib/active_support/callbacks.rb:750:in `_run_call_callbacks'
activesupport (5.0.6) lib/active_support/callbacks.rb:90:in `run_callbacks'
actionpack (5.0.6) lib/action_dispatch/middleware/callbacks.rb:36:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/remote_ip.rb:79:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/debug_exceptions.rb:49:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.6) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.6) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:69:in `block in tagged'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:69:in `tagged'
railties (5.0.6) lib/rails/rack/logger.rb:24:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.4) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.6) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.4) lib/rack/sendfile.rb:111:in `call'
railties (5.0.6) lib/rails/engine.rb:522:in `call'
puma (3.11.3) lib/puma/configuration.rb:225:in `call'
puma (3.11.3) lib/puma/server.rb:624:in `handle_request'
puma (3.11.3) lib/puma/server.rb:438:in `process_client'
puma (3.11.3) lib/puma/server.rb:302:in `block in run'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `call'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `block in spawn_thread'
Started GET "/" for 172.18.0.1 at 2018-03-25 22:01:01 +0000
  
PG::ConnectionBad (could not translate host name "db" to address: Name or service not known
):
  
pg (0.21.0) lib/pg.rb:56:in `initialize'
pg (0.21.0) lib/pg.rb:56:in `new'
pg (0.21.0) lib/pg.rb:56:in `connect'
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:671:in `connect'
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:217:in `initialize'
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `new'
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:729:in `new_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:773:in `checkout_new_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:752:in `try_to_checkout_new_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:713:in `acquire_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:490:in `checkout'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:364:in `connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:883:in `retrieve_connection'
activerecord (5.0.6) lib/active_record/connection_handling.rb:128:in `retrieve_connection'
activerecord (5.0.6) lib/active_record/connection_handling.rb:91:in `connection'
activerecord (5.0.6) lib/active_record/migration.rb:559:in `connection'
activerecord (5.0.6) lib/active_record/migration.rb:546:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/callbacks.rb:38:in `block in call'
activesupport (5.0.6) lib/active_support/callbacks.rb:97:in `__run_callbacks__'
activesupport (5.0.6) lib/active_support/callbacks.rb:750:in `_run_call_callbacks'
activesupport (5.0.6) lib/active_support/callbacks.rb:90:in `run_callbacks'
actionpack (5.0.6) lib/action_dispatch/middleware/callbacks.rb:36:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/remote_ip.rb:79:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/debug_exceptions.rb:49:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.6) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.6) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:69:in `block in tagged'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:69:in `tagged'
railties (5.0.6) lib/rails/rack/logger.rb:24:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.4) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.6) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.4) lib/rack/sendfile.rb:111:in `call'
railties (5.0.6) lib/rails/engine.rb:522:in `call'
puma (3.11.3) lib/puma/configuration.rb:225:in `call'
puma (3.11.3) lib/puma/server.rb:624:in `handle_request'
puma (3.11.3) lib/puma/server.rb:438:in `process_client'
puma (3.11.3) lib/puma/server.rb:302:in `block in run'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `call'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `block in spawn_thread'
Started GET "/" for 172.18.0.1 at 2018-03-25 22:01:19 +0000
  
ActiveRecord::NoDatabaseError (FATAL:  database "cryptopull" does not exist
):
  
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:675:in `rescue in connect'
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:671:in `connect'
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:217:in `initialize'
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `new'
activerecord (5.0.6) lib/active_record/connection_adapters/postgresql_adapter.rb:37:in `postgresql_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:729:in `new_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:773:in `checkout_new_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:752:in `try_to_checkout_new_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:713:in `acquire_connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:490:in `checkout'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:364:in `connection'
activerecord (5.0.6) lib/active_record/connection_adapters/abstract/connection_pool.rb:883:in `retrieve_connection'
activerecord (5.0.6) lib/active_record/connection_handling.rb:128:in `retrieve_connection'
activerecord (5.0.6) lib/active_record/connection_handling.rb:91:in `connection'
activerecord (5.0.6) lib/active_record/migration.rb:559:in `connection'
activerecord (5.0.6) lib/active_record/migration.rb:546:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/callbacks.rb:38:in `block in call'
activesupport (5.0.6) lib/active_support/callbacks.rb:97:in `__run_callbacks__'
activesupport (5.0.6) lib/active_support/callbacks.rb:750:in `_run_call_callbacks'
activesupport (5.0.6) lib/active_support/callbacks.rb:90:in `run_callbacks'
actionpack (5.0.6) lib/action_dispatch/middleware/callbacks.rb:36:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/remote_ip.rb:79:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/debug_exceptions.rb:49:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
railties (5.0.6) lib/rails/rack/logger.rb:36:in `call_app'
railties (5.0.6) lib/rails/rack/logger.rb:24:in `block in call'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:69:in `block in tagged'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:26:in `tagged'
activesupport (5.0.6) lib/active_support/tagged_logging.rb:69:in `tagged'
railties (5.0.6) lib/rails/rack/logger.rb:24:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/request_id.rb:24:in `call'
rack (2.0.4) lib/rack/runtime.rb:22:in `call'
activesupport (5.0.6) lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/executor.rb:12:in `call'
actionpack (5.0.6) lib/action_dispatch/middleware/static.rb:136:in `call'
rack (2.0.4) lib/rack/sendfile.rb:111:in `call'
railties (5.0.6) lib/rails/engine.rb:522:in `call'
puma (3.11.3) lib/puma/configuration.rb:225:in `call'
puma (3.11.3) lib/puma/server.rb:624:in `handle_request'
puma (3.11.3) lib/puma/server.rb:438:in `process_client'
puma (3.11.3) lib/puma/server.rb:302:in `block in run'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `call'
puma (3.11.3) lib/puma/thread_pool.rb:120:in `block in spawn_thread'
Started GET "/" for 172.18.0.1 at 2018-03-25 22:02:53 +0000
Processing by Rails::WelcomeController#index as HTML
  Parameters: {"internal"=>true}
  Rendering /bundle/gems/railties-5.0.6/lib/rails/templates/rails/welcome/index.html.erb
  Rendered /bundle/gems/railties-5.0.6/lib/rails/templates/rails/welcome/index.html.erb (3.5ms)
Completed 200 OK in 9ms (Views: 7.5ms | ActiveRecord: 0.0ms)


